collections.md

```diff
@@ -123,6 +124,26 @@ The `all` method simply returns the underlying array represented by the collecti
 
     // [1, 2, 3]
 
+<a name="method-avg"></a>
+#### `avg()` {#collection-method}
+
+The `avg` method returns the average of all items in the collection:
+
+    collect([1, 2, 3, 4, 5])->avg();
+
+    // 3
+
+If the collection contains nested arrays or objects, you should pass a key to use for determining which values to sum:
```

Comment:
Sorry, which values to avg(). Unfortunatelly, I can't fix it right now.
Em 26/09/2015 14:51, "Taylor Otwell" <notifications@github.com> escreveu:

> In collections.md
> <https://github.com/laravel/docs/pull/1812#discussion_r40496252>:
>
> > @@ -123,6 +124,26 @@ The `all` method simply returns the underlying array represented by the collecti
> >
> >      // [1, 2, 3]
> >
> > +<a name="method-avg"></a>
> > +#### `avg()` {#collection-method}
> > +
> > +The `avg` method returns the average of all items in the collection:
> > +
> > +    collect([1, 2, 3, 4, 5])->avg();
> > +
> > +    // 3
> > +
> > +If the collection contains nested arrays or objects, you should pass a key to use for determining which values to sum:
>
> Which values to sum or which values to avg?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/laravel/docs/pull/1812/files#r40496252>.
>